CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fixed a bug where a job gets listed in sys.jobs even though it has resulted in 
```

Comment:
this was actually not specific to TableUnknownException but on any parser/analyzer errors the job entry wasn't cleaned up